Update Get-CsOnlineVoiceUser.md (#9205)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineUser.md

1. LineURI set via OnPrem AD.
2. Direct Routing numbers assigned to users via Set-CsUser.

> In the Teams PowerShell Module version 3.0.0 or later, the **OnPremLineURI** attribute refers only to the LineURI that's set via OnPrem AD. Previously, **OnPremLineURI** also referred to Direct Routing numbers that were assigned to users via the Set-CsUser cmdlet. OnPremLineURIMaluallySet is now deprecated as OnPremLineURI is representative the On-Prem assignment. Also, Direct Routing numbers are available in the **LineURI** attribute. You can distinguish Direct Routing Numbers from Calling Plan Numbers by looking at the **FeatureTypes** attribute.

Reviewer: ```suggestion
In the Teams PowerShell Module version 3.0.0 or later, the **OnPremLineURI** attribute refers only to the LineURI that's set via OnPrem AD. Previously, **OnPremLineURI** also referred to Direct Routing numbers that were assigned to users via the Set-CsUser cmdlet. OnPremLineURIMaluallySet is now deprecated as OnPremLineURI is representative of the On-Prem assignment. Also, Direct Routing numbers are available in the **LineURI** attribute. You can distinguish Direct Routing Numbers from Calling Plan Numbers by looking at the **FeatureTypes** attribute.
```

> In Teams PowerShell Module version 3.0.0 and later, the **OnPremLineURI** attribute refers only to the LineURI that's set via OnPrem AD. Previously, **OnPremLineURI** also referred to Direct Routing numbers that were assigned to users via the Set-CsUser cmdlet. OnPremLineURIMaluallySet is now deprecated as OnPremLineURI is representative of the On-Prem assignment. Also, Direct Routing numbers are available in the **LineURI** attribute. You can distinguish Direct Routing Numbers from Calling Plan Numbers by looking at the **FeatureTypes** attribute.

Reviewer 2: ```suggestion
In Teams PowerShell Module version 3.0.0 and later, the **OnPremLineURI** attribute refers only to the LineURI that's set via OnPrem AD. Previously, **OnPremLineURI** also referred to Direct Routing numbers that were assigned to users via the Set-CsUser cmdlet. OnPremLineUriManuallySet is now deprecated as OnPremLineURI is representative of the On-Prem assignment. Also, Direct Routing numbers are available in the **LineURI** attribute. You can distinguish Direct Routing Numbers from Calling Plan Numbers by looking at the **FeatureTypes** attribute.
```

**Policy Attributes**:

- PolicyProperty comparison works only when "Authority" is provided in the value. For ex: `-Filter "TeamsMessagingPolicy -eq '<Authority>:<Value>'"`

> "Authority" can contain any of these two values: Host or Tenant for a policy type (Configurations that are provided by Default are referred to as Host configurations while Admin-created configurations are considered Tenant configurations)

Reviewer: ```suggestion
"Authority" can contain any of these two values: Host or Tenant for a policy type (configurations that are provided by default are referred to as Host configurations while admin-created configurations are considered Tenant configurations).
```

- Phone
- WindowsEmailAddress

> The following attributes are temporarily unavailable in the output when using the "-Filter" or when using the cmdlet without any input parameter:

Reviewer 3: Probably better to say - ...... when used without the "-Identity" parameter:

Author: Done

> The following attributes are temporarily unavailable in the output when using the "-Filter" or when using the cmdlet without any input parameter:

Reviewer: ```suggestion
The following attributes are temporarily unavailable in the output when using the "-Filter" or when used without the "-Identity" parameter:
```

- SoftDeletionTimeStamp
- WhenChanged
- OnPremHostingProvider
- OnPremSipEnabled
- OnPremSipAddress
- OnPremOptionFlags

> Note: The above attributes will be available in the near future

Reviewer: ```suggestion
**Note**: These attributes will be available in the near future.
```
